src/backend/aspen/workflows/import_locations/save.py

```diff
@@ -0,0 +1,36 @@
+import click
+
+from aspen.config.config import Config
+from aspen.database.connection import (
+    get_db_uri,
+    init_db,
+    session_scope,
+    SqlAlchemyInterface,
+)
+
+
+@click.command("save")
+@click.option("--test", type=bool, is_flag=True)
+def cli(
+    test: bool,
+):
+    if test:
+        print("Success!")
+        return
+
+    interface: SqlAlchemyInterface = init_db(get_db_uri(Config()))
+    with session_scope(interface) as session:
+        # Insert all locations from gisaid_metadata
+        insert_rows_from_gisaid = "INSERT INTO aspen.locations (region, country, division, location) SELECT DISTINCT region, country, division, location FROM aspen.gisaid_metadata ON CONFLICT (region, country, division, location) DO NOTHING"
+        # Make sure we have equivalent rows for region/country/division but with null locations
+        insert_rows_with_null_locations = "INSERT INTO aspen.locations (region, country, division) SELECT DISTINCT region, country, division FROM aspen.locations EXCEPT SELECT DISTINCT region, country, division FROM aspen.locations WHERE location IS NULL"
```

Comment:
why did you decide to use raw sql here instead of sqlalchemy api? just curious

Reply:
@jgadling do you want to take this? I know when we first reasoned about how to do this we were thinking in SQL syntax

Reply:
Know what? I just assumed SQLAlchemy didn't have a construct for `insert ... select` or `on conflict` but it turns out that it does:
https://docs.sqlalchemy.org/en/14/core/dml.html?highlight=from_select#sqlalchemy.sql.expression.Insert.from_select
https://docs.sqlalchemy.org/en/14/dialects/sqlite.html?highlight=on_conflict#sqlalchemy.dialects.sqlite.Insert.on_conflict_do_nothing

I even see an except clause:
https://docs.sqlalchemy.org/en/14/core/selectable.html?highlight=except#sqlalchemy.sql.expression.except_

I'll see about translating these queries to SA code in the morning.